mudita/mudita-center
# [PDA-694] Export calendar to file - single events

Jira: [PDA-694]

**Description**

This code adds exporting calendar events to .ics file. No repeated events support (will be added in different task/pr).


How to test:
1. Go to calendar view
2. Check some events
3. Click export in calendar panel
4. Choose location.


<details>
<summary><b>Screenshots</b></summary>
// put images here

![Zrzut ekranu 2021-02-15 o 15 23 05](https://user-images.githubusercontent.com/22203423/107958022-b6124300-6fa1-11eb-9caf-fccfb0420148.png)
</details>

**Self check**

- [x] Self CR'd
- [x] Description and screenshots attached

**PR Status**


**Blockers**

**Deploy Notes**
